### PR TITLE
[Dessert] - Notion post 

### PR DESCRIPTION
**Pull request by jhqian0909**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Post for Notion.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# How Has This Been Tested?

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/Dessert_blog/Apps/2021-06-06-Notion.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+template: overrides/blogs.html
+---
+# 功能强大，颜值在线的个人笔记应用 - Notion
+
+无论在学校还是工作中，记笔记都是让生活高效有序的有效方式。  
+常见的个人笔记软件包括：
+- 微软的**OneNote**
+- 国产的**印象笔记**
+
+今天将要介绍一款ios扁平风的宝藏App - [Notion](https://www.notion.so/)。
```
Comment (Realvincentyuan) [inline docs/Dessert_blog/Apps/2021-06-06-Notion.md:11]: @jhqian0909 ， iOS的字母大小写需要修正。

File: docs/Dessert_blog/Apps/2021-06-06-Notion.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+普通笔记确实很普通，可以满足大部分用户的基础需求。但在细节方面有点乏善可陈，比如代码块中缺少代码高亮，文字编辑也没有格式刷或者纯文本格式等实用的小细节。  
+另外，宣传的团队协作功能推出不久，并没有太多的存在感。  
+## 3. Notion  
+Notion是硅谷初创公司研发的一款笔记应用。官网给人的第一印象是美观简洁的插画风格，可以看出团队是在用心做产品的。 
+<figure>
+  <img src="https://cdn.jsdelivr.net/gh/BulletTech2021/Pics/2021-6-6/1622968071519-notion1.png" width="600" />
+  <figcaption>Notion</figcaption>
+</figure>
+
+口号主打的是团队协作的工作空间。收费方面，免费的个人账户基本能够满足个人使用的需求。Notion支持移动端，网页端以及本地客户端。
```
Comment (Realvincentyuan) [inline docs/Dessert_blog/Apps/2021-06-06-Notion.md:27]: 这句话只提到了免费的个人账户， 可以来张收费的图介绍一下其他类型的账号吗？

File: docs/Dessert_blog/Apps/2021-06-06-Notion.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+本文是一篇个人向的点评，从实际使用及个人偏好出发，仅代表个人观点。  
+## 1. OneNote  
+作为微软出品，它有着一如既往的Office风格，说不上丑也说不上好看。可个人使用，也可作为企业团队协作或知识共享的平台。功能不算强大，算是一款无功无过的企业级标准产品。  
+## 2. 印象笔记（Evernote）  
+作为一款国产个人笔记产品，知名度较高，受众较大。但是一打开主页就来了一波“会员5折买3年”的广告，不免有点“印象扣分”。    
+笔记分为两种：普通的和超级笔记。超级笔记和Notion惊人的相似，都是由一个个模块（Block）构成，如代码块，表格等。但令人窒息的是不少模块都是付费账号专享。  
+普通笔记确实很普通，可以满足大部分用户的基础需求。但在细节方面有点乏善可陈，比如代码块中缺少代码高亮，文字编辑也没有格式刷或者纯文本格式等实用的小细节。  
+另外，宣传的团队协作功能推出不久，并没有太多的存在感。  
+## 3. Notion  
+Notion是硅谷初创公司研发的一款笔记应用。官网给人的第一印象是美观简洁的插画风格，可以看出团队是在用心做产品的。
```
Comment (Realvincentyuan) [inline docs/Dessert_blog/Apps/2021-06-06-Notion.md:21]: 介绍Notion的部分，建议添加一些小标题，这样在blog里比较好对应。另外，我的印象中，mkdocs里不同样式的内容中间要空行，这个可以merge之后再看咯。

@jhqian0909

**Review by Realvincentyuan — COMMENTED** [review]
总体我觉得挺好的，有些结构和格式方面的小建议

File: docs/Dessert_blog/Apps/2021-06-06-Notion.md (hunk 1 above)
Comment (jhqian0909) [reply docs/Dessert_blog/Apps/2021-06-06-Notion.md]: changed

File: docs/Dessert_blog/Apps/2021-06-06-Notion.md (hunk 2 above)
Comment (jhqian0909) [reply docs/Dessert_blog/Apps/2021-06-06-Notion.md]: 加了张图

File: docs/Dessert_blog/Apps/2021-06-06-Notion.md (hunk 3 above)
Comment (jhqian0909) [reply docs/Dessert_blog/Apps/2021-06-06-Notion.md:28]: 加了小标题以及空行

File: docs/Dessert_blog/Apps/2021-06-06-Notion.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+<figure>
+  <img src="https://cdn.jsdelivr.net/gh/BulletTech2021/Pics/2021-6-6/1622968058825-notion4.png" width="600" />
+  <figcaption>Personal Home</figcaption>
+</figure> 
+
+当然，我们也可以自制模板，便于调用。
+
+### 3.5 团队协作
+
+我们也尝试过Notion的团队协作功能，除了可以共享笔记，知识分享外。我们也可以看到实时的更新、评论或利用看板，满足部分项目管理的需求。
```
Comment (jz263) [inline docs/Dessert_blog/Apps/2021-06-06-Notion.md:81]: 标点符号：知识分享外，我们也可以

File: docs/Dessert_blog/Apps/2021-06-06-Notion.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+</figure> 
+
+一应俱全的**个人主页**等等:  
+
+<figure>
+  <img src="https://cdn.jsdelivr.net/gh/BulletTech2021/Pics/2021-6-6/1622968058825-notion4.png" width="600" />
+  <figcaption>Personal Home</figcaption>
+</figure> 
+
+当然，我们也可以自制模板，便于调用。
```
Comment (Realvincentyuan) [inline docs/Dessert_blog/Apps/2021-06-06-Notion.md:77]: 我个人期待这里稍微加点demo或者饮用一些notion自带的图片或者教程 ：p

**Comment by Realvincentyuan** [comment]
@jhqian0909 

文章头部请加入作者信息

`
!!! info 作者：原子弹起飞，发布于2021年6月3日，阅读时间：约10分钟，微信公众号文章链接：:fontawesome-solid-link:
`

**Review by Realvincentyuan — APPROVED** [review]
LGTM

**Pull request merged**